src/app/static/src/app/components/modelOutput/ModelOutput.vue

```diff
@@ -18,7 +18,8 @@
             </div>
 
             <div id="barchart-container" :class="selectedTab==='Bar' ? 'col-md-12' : 'd-none'">
-                <barchart :chartdata="chartdata" :filters="barchartFilters" :indicators="barchartIndicators"></barchart>
+                <barchart :chartdata="chartdata" :filters="barchartFilters" :indicators="barchartIndicators"
+                          :selections="barchartSelections" v-on:change-selections="updateBarchartSelections"></barchart>
```

Comment:
I see why you've gone for a custom event name over "change", but "change-selections" is quite verbose - how about "select"?

Reply:
Not that keen on `select` as it's quite associated with selecting from a drop-down. I could go with `update`?

Reply:
Ah yeah fair point. "update" seems good :+1: